docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -217,6 +279,12 @@ Go over the output tensors and process the inference results.
 
 @endsphinxtab
 
+@sphinxtab{C}
+
+@snippet docs/snippets/src/main.c part6
+
+@endsphinxtab
+
 @endsphinxtabset
 
 ## Step 7. Link and Build Your Application with OpenVINO™ Runtime (example)
```

Comment:
The step 7 should be extended for c api too

Reply:
done